anonymizer_gui.py

```diff
@@ -174,4 +175,5 @@ def zipDicom(self):
 if __name__ == "__main__":
     root = Tk()
     app = dicom_anonymizer(root)
+    root.lift()
```

Comment:
This one line looks like it does something different from the rest of the pull request and can probably be moved to a different pull request (I think it just raises the window to the top.), to make this one a little easier to read